File: aula03/abstracao.py
class Carro():
    def __init__(self, marca, modelo):
        self.marca = marca
        self.modelo = modelo
        self.ligado = False

    def ligar_motor(self):
        self.ligado = True

    def desligar_motor(self):
        self.ligado = False

c1 = Carro("vw", "gol")
print(c1.ligado)
c1.ligar_motor()
print(c1.ligado)

File: aula03/abstracao.js
class Carro {
    constructor(marca, modelo) {
        this.marca = marca;
        this.modelo = modelo;
        this.ligado = false;
    }

    ligar_motor() {
        this.ligado = true;
    }

    desligar_motor() {
        this.ligado = false;
    }
}

const c1 = new Carro("VW", "Gol");

console.log(`Modelo: ${c1.modelo}, Marca: ${c1.marca}, Ligado: ${c1.ligado}`);

c1.ligar_motor();

console.log(`Modelo: ${c1.modelo}, Marca: ${c1.marca}, Ligado: ${c1.ligado}`);
